src/pages/index.js

```diff
@@ -17,47 +42,111 @@ popupImage.setEventListeners();
 const openPopupImage = (data) => {
   popupImage.open(data);
 };
-const renderCard = (card) => {
-  cardRenderer.addItem(createCard(card));
+
+const deletePopup = new PopupWithForm(constants.deletePopup, (data) => {
+  api
+    .deleteCard(data._id, data)
+    .then(() => {
+      deletePopup.submitButton.textContent = "Delete Card";
```

Comment:
`      deletePopup.submitButton.textContent = "Delete Card";
`

You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.

**Please, correct it everywhere**